src/inmanta/server/agentmanager.py

```diff
@@ -165,16 +165,14 @@ def get_agent_client(self, tid: uuid.UUID, endpoint: str) -> Optional[ReturnClie
 
     @gen.coroutine
     def start(self) -> NoneGen:
-        self.running = True
+        yield super().start()
```

Comment:
The running variable of AgentManager should be removed. It shields the running variable of the ServerSlice class.